### PR TITLE
Add English README

### PR DESCRIPTION
**Pull request by Nicole00**

### CODE REVIEW

File: nebula-exchange/README.md [hunk 1]
```diff
@@ -1,53 +1,44 @@
...
-关于 Exchange 2.0 的详细使用说明，请参考 Exchange 1.0 的使用文档[《Nebula Exchange 用户手册》](https://docs.nebula-graph.com.cn/nebula-exchange/about-exchange/ex-ug-what-is-exchange/ "点击前往 Nebula Graph 网站")。
-
-特性 & 注意事项：
-
-*1. Nebula Graph 2.0 暂时只支持 String 类型的 VID，请勿配置点 ID 或者边的 src_id、dst_id 的 policy。*
-
-*2. Exchange 2.0 新增 null、Date、DateTime、Time 类型数据的导入。*
-
-*3. Exchange 2.0 支持 Hive on Spark 以外的 Hive 数据源，需在配置文件中配置 Hive 源，具体配置示例参考 [server_application.conf](https://github.com/vesoft-inc/nebula-spark-utils/blob/main/nebula-exchange/src/main/resources/server_application.conf) 中 Hive 的配置。*
+    After the packaging, you can see the newly generated nebula-exchange-2.0.0.jar under this directory nebula-spark-utils/nebula-exchange/target/.
```
Comment (oldLady344) [inline nebula-exchange/README.md:30]: under the nebula-spark-utils/nebula-exchange/target/ directory.

File: nebula-exchange/README.md [hunk 2]
```diff
@@ -1,53 +1,44 @@
...
+    
+## How to use
 
-*4. Exchange 2.0 将导入失败的 INSERT 语句进行落盘，存于配置文件的 error/output 路径中。*
+For more detail about Exchange, please refer to [Exchange 1.0](https://github.com/vesoft-inc/nebula-java/tree/v1.0/tools/exchange) .
```
Comment (oldLady344) [inline nebula-exchange/README.md:34]: For more details

File: nebula-exchange/README.md [hunk 3]
```diff
@@ -1,53 +1,44 @@
...
-*5. 配置文件参考 [application.conf](https://github.com/vesoft-inc/nebula-spark-utils/tree/main/nebula-exchange/src/main/resources)。*
 
-## 贡献
+## New Features
 
-Nebula Exchange 2.0 是一个完全开源的项目，欢迎开源爱好者通过以下方式参与：
+1. Support to import VID of String type
+2. Support to import data of Null, Date, DateTime, and Time types
+3. Support to import data from other Hive sources than Hive on Spark
+4. Added record and retry of the INSERT statement failure during the import
```
Comment (oldLady344) [inline nebula-exchange/README.md:42]: 1. Supports importing vertex data with String type IDs.
2. Supports importing data of the Null, Date, DateTime, and Time types
3. Supports importing data from other Hive sources than Hive on Spark
4. Supports records and retry of the INSERT statement after a failure during data import

File: nebula-exchange/README.md [hunk 4]
```diff
@@ -1,53 +1,44 @@
...
 
-- 前往 [Nebula Graph 论坛](https://discuss.nebula-graph.com.cn/ "点击前往“Nebula Graph 论坛") 上参与 Issue 讨论，如答疑、提供想法或者报告无法解决的问题
-- 撰写或改进文档
-- 提交优化代码
+Refer to [application.conf](https://github.com/vesoft-inc/nebula-spark-utils/tree/main/nebula-exchange/src/main/resources) to see how to config the configuration file.
```
Comment (oldLady344) [inline nebula-exchange/README.md:44]: see how to config the configuration file. --> as an example to edit the configuration file.

File: nebula-exchange/README.md (hunk 3 above)
Comment (Nicole00) [reply nebula-exchange/README.md:42]: 3. from other Hive sources

Comment (oldLady344) [reply nebula-exchange/README.md]: > 1. 是增加了对非Hive on Spark的支持，但同时也是支持Hive on Spark的。

Supports importing data from other Hive sources besides Hive on Spark.

**Review by oldLady344 — APPROVED** [review]

**Pull request merged**